recipe/meta.yaml

```
@@ -26,14 +26,14 @@ requirements:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - cmake >=3.18
-      # We currently don't have ninja 1.11, which upstream specifies. 
+      # We currently don't have ninja 1.11, which upstream specifies.
       # However, this feedstock builds and passes tests with v1.10.2
     - ninja >=1.10.2  # [unix]
   host:
-    # This basically forces to pick scipy (and any other suitable packages in the chain, like numpy) 
-    # that was built with openblas rather than mkl. Openblas typically pairs with llvm-openmp and mkl with intel-openmp on osx-64. 
-    # By default, mkl variants are picked. By picking the variant that uses openblas rather 
-    # than mkl, intel-openmp will not be pulled in and so you will not have llvm-openmp / intel-openmp conflict. 
+    # This basically forces to pick scipy (and any other suitable packages in the chain, like numpy)
+    # that was built with openblas rather than mkl. Openblas typically pairs with llvm-openmp and mkl with intel-openmp on osx-64.
+    # By default, mkl variants are picked. By picking the variant that uses openblas rather
+    # than mkl, intel-openmp will not be pulled in and so you will not have llvm-openmp / intel-openmp conflict.
     - blas =*=openblas  # [osx and x86_64]
     - scikit-build-core >=0.9.3
```

```suggestion
    - scikit-build-core >=0.10.1
```
https://github.com/microsoft/LightGBM/blob/v4.6.0/python-package/pyproject.toml#L61